cmd/enumerate_github/main.go

```diff
@@ -151,6 +154,26 @@ func searchWorker(s *githubsearch.Searcher, logger *zap.Logger, queries, results
 	}
 }
 
+func writeMarker(ctx context.Context, markerFile, outFile string) (err error) {
+	marker, e := cloudstorage.NewWriter(ctx, markerFile)
+	if e != nil {
+		return fmt.Errorf("open marker: %w", err)
+	}
+	defer func() {
+		if e := marker.Close(); e != nil {
+			// Return the error using the named return value if it isn't
+			// already set.
+			if err == nil {
+				err = fmt.Errorf("closing marker: %w", err)
```

Comment:
should the second `err` here be `e` instead? ****

Reply:
Fixed